src/webui/www/private/scripts/client.js

```diff
@@ -258,7 +294,13 @@ let selectedStatus = localPreferences.get("selected_filter", "all");
 let setStatusFilter = () => {};
 let toggleFilterDisplay = () => {};
 
-window.addEventListener("DOMContentLoaded", (event) => {
+window.addEventListener("DOMContentLoaded", async (event) => {
+    await window.qBittorrent.Client.initializeClientData();
+    window.qBittorrent.ColorScheme.update();
+
+    useAutoHideZeroStatusFilters = clientData.get("hide_zero_status_filters") === true;
+    displayFullURLTrackerColumn = clientData.get("full_url_tracker_column") === true;
+
     window.qBittorrent.LocalPreferences.upgrade();
```

Comment:
`window.qBittorrent.LocalPreferences.upgrade();` is intended to be executed first in this function.